administrative-sdk/connection.js

```diff
@@ -192,32 +193,34 @@ class Connection {
    * @param {callback} [ecb] The callback that handles the error response.
```

Comment:
Missing documentation update. Callbacks are no longer taken and a promise is returned.